keyvalue/clients/src/main/java/tachyon/client/keyvalue/KeyValueStore.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client.keyvalue;
+
+import java.io.IOException;
+
+import tachyon.TachyonURI;
+import tachyon.annotation.PublicApi;
+import tachyon.exception.TachyonException;
+
+/**
+ * Client to access a key-value store in Tachyon.
+ */
+@PublicApi
+public interface KeyValueStore {
```

Comment:
here I try to keep the naming aligned with filesystem API, where we use  `FileSystem` as the interface for fs operations.